.bazelignore

```diff
@@ -1 +1,2 @@
 examples
+bazel-rules_ros2
```

Comment:
I guess this is for debugging. Please rm.

Reply:
You're right -- This is needed to make the Starlark debugger in VS Code work, see https://github.com/bazelbuild/bazel/issues/10653#issuecomment-694230015
it's of course unrelated to the actual changes in the PR, so I removed it again